tools/walletextension/common/common.go

```diff
@@ -99,3 +99,23 @@ func NewFileLogger() gethlog.Logger {
 
 	return logger
 }
+
+// GetBestFormat returns the best format for a message based on available formats that are supported by the user
+func GetBestFormat(formatsSlice []string) viewingkey.SignatureType {
```

Comment:
this should live in the same file with the other message generator stuff

Reply:
moved